packit.yaml

```diff
@@ -1,4 +1,6 @@
 upstream_project_url: https://github.com/cockpit-project/cockpit-podman
+# enable notification of failed downstream jobs as issues
+issue_repository: https://github.com/cockpit-project/cockpit-podman
```

Comment:
Hmm, lets see how verbose this is gonna be :)

Reply:
On failures you'll get something like https://github.com/martinpitt/umockdev/issues/178 . But ideally of course it should be silent :grinning: